kits/beta-base/test/docker.js

```diff
@@ -38,10 +38,10 @@ contract('Beta Base Kit', accounts => {
         daoAddress = getEventResult(daoReceipt, 'DeployInstance', 'dao')
         const tokenAddr = getEventResult(daoReceipt, 'DeployInstance', 'token')
         // generated Voting app
-        financeAddress = getAppProxy(daoReceipt, appIds[0])
-        tokenManagerAddress = getAppProxy(daoReceipt, appIds[1])
-        vaultAddress = getAppProxy(daoReceipt, appIds[2])
-        votingAddress = getAppProxy(daoReceipt, appIds[3])
+        financeAddress = getAppProxy(daoReceipt, appIds[1])
```

Comment:
I know it's weird, but given that we are reusing the same deployment scripts we have for the beta-base kit, I needed to declare the `Agent` app there as well. The beta-base deployment script deploy the base implementation of each app and registers them in the APM. Thus, we have a new app id here we hadn't before. I know it's a little bit sketchy but otherwise I would have had to copy-paste all the beta-base deployment scripts or refactor them which was huge task exceeding this PR. However, we have already started working on a refactor to polish and re-structure the way we share code between these kits making it more configurable and less cumbersome.